domain/src/main/assets/drag_and_drop_test_exploration.json

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
Are these json files only meant to be used for unit tests, or are we using it in the app as well?
If we want it only for unit tests we should move it to domain/src/test/... @BenHenning

Reply:
I think this one wasn't answered @aggarwalpulkit596 ?

Reply:
This was meant for ben i guess ?

Reply:
I think we'll want to use this for UI-facing tests, too, and probably for manual testing. I think it makes sense to keep it here. We'll eventually change asset handling such that the dev/test-only assets aren't included in prod builds.

Reply:
@vinitamurthi please resolve this thread if you're satisfied with Ben's answer